src/screens/Feed.js

```diff
@@ -5,18 +5,39 @@ import {
   StyleSheet,
   FlatList,
   TouchableWithoutFeedback,
+  Text,
+  SafeAreaView,
+  Alert
 } from 'react-native';
+
 import SettingsButtonHorizontal from './app/Components/SettingsButtonHorizontal/SettingsButtonHorizontal';
 import ProfileAvatar from './app/Components/ProfileAvatar/ProfileAvatar';
 import Post from './app/Components/Post/Post';
+import Swiper from 'react-native-swiper';
+import { Divider } from 'react-native-paper';
+import * as ScreenshotDetector from 'react-native-screenshot-detect';
+
+
 export default class FeedScreen extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      swiperIndex: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.eventEmitter = ScreenshotDetector.subscribe(() => {
+      Alert.alert('The post with caption', posts[this.state.swiperIndex].caption, 'was screenshotted');
```

Comment:
We can come up with a better alter message, Possibly username screenshot  + smaller image + caption.
 Or maybe no image